shell/install.sh

```diff
@@ -358,8 +366,8 @@ if [ -n "$RESTORE" ]; then
         exit 1
     fi
     if [ "$NOBACKUP" = 1 ]; then
-        printf "## This will clear $OPAM and $OPAMROOT. Continue ? [Y/n] "
-        read R
+        prompt "## This will clear $OPAM and $OPAMROOT. Continue ? [Y/n] "
+        R=$?
```

Comment:
unless I misread something I believe `$?` is always a number
```suggestion
```

Reply:
You're correct. This one was an oversight from before I understood that :see_no_evil: